### PR TITLE
Fix build error on Arm64

### PR DESCRIPTION
**Pull request by cyb70289**

### CODE REVIEW

File: include/xsimd/arch/xsimd_neon64.hpp [hunk 1]
```diff
@@ -658,7 +658,7 @@ namespace xsimd
             {
                 if (n == I)
                 {
-                    return vextq_f64(lhs, rhs, n);
+                    return vextq_f64(lhs, rhs, I);
```
Comment (cyb70289) [inline include/xsimd/arch/xsimd_neon64.hpp:661]: last augment of `vextq` must be constant

**Comment by cyb70289** [comment]
cc @JohanMabille @pitrou

**Comment by pitrou** [comment]
Again, it seems weird that the ARM CI checks didn't catch this? (I see at least one named "Linux_1 gcc_4_aarch64")

**Comment by JohanMabille** [comment]
Thanks!

**Pull request merged**